lib/quicly.c

```diff
@@ -21,6 +21,9 @@
  */
 #include <assert.h>
 #include <inttypes.h>
+#if defined(__FreeBSD__)
+#include <sys/types.h>
+#endif
```

Comment:
Thank you for spotting this. Let's include the file always.

```suggestion
#include <sys/types.h>
```